build/android-cpu_features.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [[ -z ${ANDROID_NDK_ROOT} ]]; then
+    echo -e "(*) ANDROID_NDK_ROOT not defined\n"
+    exit 1
+fi
+
+if [[ -z ${ARCH} ]]; then
+    echo -e "(*) ARCH not defined\n"
+    exit 1
+fi
+
+if [[ -z ${API} ]]; then
+    echo -e "(*) API not defined\n"
+    exit 1
+fi
+
+# ENABLE COMMON FUNCTIONS
+. ${BASEDIR}/build/android-common.sh
+
+# PREPARE PATHS & DEFINE ${INSTALL_PKG_CONFIG_DIR}
+LIB_NAME="cpu_features"
+set_toolchain_clang_paths ${LIB_NAME}
+
+PREFIX=${BASEDIR}/prebuilt/android-$(get_target_build)/${LIB_NAME}
+
+android_cmake() {
+  local cmake=$(which cmake)
+if [[ -z ${cmake} ]]; then
+  cmake=$(find ${ANDROID_HOME}/cmake -path \*/bin/cmake -type f -print -quit)
+fi
+if [[ -z ${cmake} ]]; then
+    echo -e "(*) cmake not found\n"
+    exit 1
+fi
+echo ${cmake}
+}
+
+create_cpu_features_package_config() {
```

Comment:
All custom package config functions are created inside `android-common.sh`. Is there a reason to define it here?

Reply:
I thought that handling all **cpu_features** related logic in single script would be cleaner. I don't see any advantage in keeping it in `android-common.sh`.

Reply:
Well, it is a design decision taken to keep library build scripts short and easy to understand. You can see that none of those scripts include function definitions.

Reply:
as you wish